routes/routes.js

```diff
@@ -0,0 +1,39 @@
+require('strict-mode')(function () {
+  const controllers = require('../controllers');
+
+const Router = require('express').Router;
+const router = new Router();
+
+
+router.get('/', (req, res) => {
+  res.json({ message: 'Welcome to krishna API!' });
+});
+
+  // ...args is not working. hence had to use req, res, next
+  router.route('/user')
+      .get((req, res, next) => controllers.user.find(req, res, next))
+      .post((req, res, next) => {
+        console.log(`req ${req}  req.body ${JSON.stringify(req.body)}`);
+        controllers.user.create(req, res, next)});
+
+  router.route('/user/:id')
+      .put((req, res, next) => controllers.user.update(req, res, next))
```

Comment:
.put(controllers.user.update) // is enough, you don't need to explicitly writing our parameters